com.scanoss.sbom-workbench.yml

```diff
@@ -13,9 +13,7 @@ finish-args:
   - "--socket=pulseaudio"
   - "--share=network"  
   - "--device=dri"    
-  - "--persist=.scanoss"
-  - "--persist=scanoss-workspace"
-  - "--persist=.pki"
+  - "--filesystem=host:create"
```

Comment:
what does `host:create` mean? Also you mention `home` but put `host`.

Reply:
thanks for catching it